### DISCUSSION TITLE
Sunbird set up

### DISCUSSION THREAD

**Discussion by Sush2553** [Sunbird-Ed/Community#334] [Sunbird ED configuration discussion]
Hello,
We are in the process of setting up sunbird Ed and reusing its services,we are looking for the exact repositories which are being used.Could you please help us with the relevant details for the same and provide links to all the repositories which are currently being used?

**Comment by beepdot** [comment]
Please see here - https://ed.sunbird.org/learn/readme

**Comment by Krishnaj20** [comment]
@Sush2553 - Pls start with the ED microsite: https://ed.sunbird.org/learn/readme . This microsite has the required details about the repos. If you have any specific questions after going thru the website please post in this thread.

@vinukumar-vs

**Comment by ranvsin2** [comment]
we were going through this documentation link https://lern.sunbird.org/learn/product-and-developer-guide and found that it takes us to this github link repositories link https://github.com/Sunbird-Lern/sunbird-lms-service for user-org service. However, in this list of github repos https://ed.sunbird.org/use-1/source-code/platform , this particular repo has been mentioned as depreciated ; and there is a different repo for user management https://github.com/project-sunbird/sunbird-user-service.
we are confused on which one should we use for user and org services.

**Reply by Krishnaj20** [reply]
@gandham-santhosh @vinukumar-vs - Please check and respond to this thread.

**Reply by vinukumar-vs** [reply]
@Sush2553 Refer this document for sunbird platform for all the dependent BB's services & its repos

**Reference Apps:** https://ed.sunbird.org/use-1/source-code/reference-apps
**Platform:** https://ed.sunbird.org/use-1/source-code/platform

@Krishnaj20 @rajeevsathish

**Reply by Krishnaj20** [reply]
@ranvsin2 - is your query answered?

**Reply by chitranshu-keshav** [reply]
@ranvsin2 @Sush2553  Reminder to mark this thread as answered
